code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -430,114 +428,70 @@ components:
                 type: array
                 items:                          
                   type: object
                   required:
                     - interfaceId
                     - protocol
-                    - port
-                    - network
-                    - interfaceName
+                    - extPort
                     - visibilityType
                   properties:
                     interfaceId:
                       type: string
-                      description:  Each Port and corresponding traffic protocol exposed by the component is identified by a name. 
-                                            Application client on user device requires this to uniquley idenify the interface. 
-                    network:
-                      type: string
-                      description: Name of the network.  In case the application has to be assoisated with more then 1 network then app provider
-                                          must define the name of the network on which this interface has to be exposed.  This parameter is required only if 
-                                          the port has to be exposed on a specific network other then default.
+                      pattern: ^[A-Za-z][A-Za-z0-9_]{3,31}$
+                      description:  Each Port and corresponding traffic protocol exposed by the component is identified by a name. Application client on user device requires this to uniquley idenify the interface. 
                     protocol:
                       type: string
                       enum:
                         - TCP
                         - UDP
                         - ANY
                       description: Defines the IP transport communication protocol i.e., TCP, UDP or ANY
-                    port:
+                    extPort:
                       type: integer
                       format: int32
                       minimum: 1
                       maximum: 65535
-                      description: Port number exposed by the component. OP may generate a dynamic port towards the UCs 
-                                          corresponding to this internal port and forward the client traffic from dynamic port to containerPort.
-                    interfaceName:
-                      type: string
-                      description:  Interface Name. Required only if application has to be attatched to a network other then default.
+                      description: Port number exposed by the component. OP may generate a dynamic port towards the UCs corresponding to this internal port  and forward the client traffic from dynamic port to containerPort.
```

Comment:
Naming and comment here is confusing, the field is called "extPort" (external port?) but the comment refers to it as both "internal port" and "containerPort", the latter of which doesn't make sense for VM based deployments. Also "UC" is not defined anywhere, so I'm not sure what that stands for.

I would suggest to rename the field back to "port", and have the comment be:
```
Port number exposed by the component. OP may generate a dynamic port towards the component instance which forwards external traffic to the component port.
```

Reply:
Agree with the observation. Will provide a change for it.

Reply:
Changes done and committed to PR